apps/discord-bot/src/commands/leaderboards/historical-leaderboard.argument.ts

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
Delete this file, it serves no purpose since `player-leaderboard.argument.ts` already has the historical autocomplete implementation.